### PR TITLE
feature - create tabbar controller

### PR DESCRIPTION
**Pull request by at-haivo**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: Youtube/Controllers/Base/AppDelegate.swift [hunk 1]
```diff
@@ -25,8 +25,46 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
         window = UIWindow(frame: UIScreen.main.bounds)
         window?.makeKeyAndVisible()
+        window?.rootViewController = createTabbarController()
+        UIApplication.shared.statusBarStyle = .lightContent
         return true
     }
+
+    // add create tabbar comtroller
+
+    func createTabbarController() -> UITabBarController {
```
Comment (at-daiho) [inline Youtube/Controllers/Base/AppDelegate.swift]: private func

File: Youtube/Controllers/Base/AppDelegate.swift [hunk 2]
```diff
@@ -25,8 +25,46 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+        tabbarController.tabBar.tintColor = UIColor.white
+        tabbarController.tabBar.barTintColor = Color.black
+
+        return tabbarController
+    }
+
+    // selection controller
+
+    func selectionController(controller: UIViewController, image: String, title: String) -> UIViewController {
+        let homeNC = NavigationController(rootViewController: controller)
```
Comment (at-daiho) [inline Youtube/Controllers/Base/AppDelegate.swift]: Why variable name is homeNC?

File: Youtube/Controllers/FavoriteVC/FavoriteViewController.swift [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+//  Copyright © 2017 AsianTech. All rights reserved.
+//
+
+import UIKit
+
+class FavoriteViewController: BaseViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = "Favorite"
```
Comment (at-daiho) [inline Youtube/Controllers/FavoriteVC/FavoriteViewController.swift]: Please remove this line

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Comment by at-haivo** [comment]
@at-daiho please review and check it

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Pull request merged**